src/app/faqs/faq-list/faq-list.component.html

```diff
@@ -0,0 +1,154 @@
+<div class="d-flex row-hl">
+  <!-- Category Selector -->
+  <div class="p-2 w-20">
+    <div *ngFor="let faqCategory of faqCategories">
+      <button
+        mat-raised-button
+        [color]="filterFaqCategory?.id == faqCategory.id ? 'accent' : ''"
+        class="card mb-3 w-100"
+        (click)="onClickCategory(faqCategory)"
+      >
+        <div
+          class="d-flex align-items-center justify-content-between flex-row p-3"
+        >
+          <i class="fa mr-4" [ngClass]="faqCategory.icon"></i>
+          <div class="d-flex flex-column align-items-center w-100">
+            <mat-card-title>
+              {{ faqCategory.title }}
+            </mat-card-title>
+            <span class="font-weight-normal line-height-normal text-wrap">
+              {{ faqCategory.description }}
+            </span>
+          </div>
+        </div>
+      </button>
+    </div>
+    <button
+      *ngIf="userIsAdmin"
+      mat-raised-button
+      color="primary"
+      class="w-100"
+      (click)="onClickAddCategory()"
+    >
+      Add Category
+    </button>
+  </div>
+
+  <!-- Categories -->
+  <div class="p-2 w-60 d-flex flex-column align-items-center">
```

Comment:
i think it should be w-80 not w-60

Reply:
I thought it looked kind of weird because then the faq section was not centered on the screen anymore. 

If we do it like that, where should the main h1 be centered, on the page or on the faq section?

Reply:
re: h1 - centered on the page is good - whatever looks good to you